docs/src/pages/components/Chips/WebChipsPage.js

```diff
@@ -231,6 +256,17 @@ const components = [
       <Paragraph>
         Dismissible chips are useful when applying filters. They are designed to
         disappear when pressed, so they cannot be toggled on and off.
+        <br />
+        <br />
+        Note that if chips are being added or removed dynamically (like in the
+        example below), you should render a{' '}
+        <BpkLink href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions">
+          <BpkCode>aria-live</BpkCode>
+        </BpkLink>{' '}
+        container describing the updates taking place. By doing this, assistive
+        technologies will know to inform the user when changes are made. By
+        default, only additions to the container will be read out, so we
```

Comment:
```suggestion
        default, only additions to the container will be read out, so you
```

For consistency, as the paragraph begins in second person (you) then switches to first (we).